### PR TITLE
Fixing up Appveyor errors for AFS docs

### PR DESCRIPTION
**Pull request by ailae**

### CODE REVIEW

File: api-reference/beta/resources/projectrome_activity.md [hunk 1]
```diff
@@ -21,19 +21,24 @@ Your user activities will be showcased in Cortana and Windows Timeline user expe
...
-|activationUrl | String | Required. URL used to launch the activity in the best native experience represented by the appId. Might launch a web-based app if no native app exists.|
-|fallbackUrl | String | Optional. URL used to launch the activity in a web-based app, if available.|
-|contentUrl | String | Optional. Used in the event the content can be rendered outside of a native or web-based app experience (for example, a pointer to an item in an RSS feed).|
-|visualElements| [visualInfo](../resources/projectrome_visualinfo.md) | Required. The object containing information to render the activity in the UX.|
-|contentInfo | Untyped JSON object | Optional. A custom piece of data - JSON-LD extensible description of content according to [schema.org](http://schema.org) syntax.|
-|historyItems | [historyItem](../resources/projectrome_historyitem.md) collection | Optional. NavigationProperty/Containment; navigation property to the activity's historyItems.|
+|userTimezone | String | Optional; the timezone in which the user's device used to generate the activity was located at activity creation time; values supplied as Olson IDs in order to support cross-platform representation|
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/projectrome_activity.md]: Why were the items in this table changed?

Comment (ailae) [reply api-reference/beta/resources/projectrome_activity.md]: I think this happened when I pulled the upstream branch. Fixed.

File: api-reference/beta/resources/projectrome_activity.md [hunk 2]
```diff
@@ -33,7 +33,12 @@ Your user activities will be showcased in Cortana and Windows Timeline user expe
 |contentUrl | String | Optional. Used in the event the content can be rendered outside of a native or web-based app experience (for example, a pointer to an item in an RSS feed).|
 |visualElements| [visualInfo](../resources/projectrome_visualinfo.md) | Required. The object containing information to render the activity in the UX.|
 |contentInfo | Untyped JSON object | Optional. A custom piece of data - JSON-LD extensible description of content according to [schema.org](http://schema.org) syntax.|
-|historyItems | [historyItem](../resources/projectrome_historyitem.md) collection | Optional. NavigationProperty/Containment; navigation property to the activity's historyItems.|
+
+## Relationships
+
+|Relationship | Type | Description|
+|:------------|:-----|:-----------|
+| historyItems| [historyItem](../resources/projectrome_historyitem.md) collection | Optional; NavigationProperty/Containment; navigation property to the activity's historyItems|
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/projectrome_activity.md]: the description of this relationship should be the same as it had been in the previous table.

File: api-reference/beta/resources/projectrome_activity.md [hunk 3]
```diff
@@ -64,6 +70,9 @@ Here is a JSON representation of the resource.
     "activationUrl": "String (URL)",
     "contentUrl": "String (URL)",
     "fallbackUrl": "String (URL)",
+    "createdDateTime": "DateTimeOffset",
+    "lastModifiedDateTime": "DateTimeOffset",
+    "id": "String",
     "contentInfo": {
         // JSON object
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/projectrome_activity.md]: the "// JSON object" statements should be removed as this is not valid JSON

File: api-reference/beta/resources/projectrome_activity.md (hunk 2 above)
Comment (ailae) [reply api-reference/beta/resources/projectrome_activity.md]: Fixed.

File: api-reference/beta/resources/projectrome_activity.md (hunk 3 above)
Comment (ailae) [reply api-reference/beta/resources/projectrome_activity.md]: Sure, done

**Pull request merged**